src/index.js

```diff
@@ -142,7 +147,149 @@ const submitFormButton = document.getElementById('submitForm');
+});
+
+radioWeb3.addEventListener('click', (event) => {
+  handleLibraryChange(event);
+});
+
+function initializeLibrarySwitches() {
+  if (libraryInUse === 'ethers') {
+    radioEthers.checked = true;
+  } else {
+    radioWeb3.checked = true;
+  }
+}
+
+function handleLibraryChange(event) {
+  libraryInUse = event.target.value;
+  if (libraryInUse === 'web3') {
+    // Deployed piggy bank smart contract with web3 library
+    piggyBankContract = web3PiggyBankContract;
+    // Deployed hst smart contract with web3 library
+    hstContract = web3HstContract;
+    // Deployed collectibles smart contract with web3 library
+    collectiblesContract = web3CollectiblesContract;
+    // Deployed failing smart contract with web3 library
+    failingContract = web3FailingContract;
+  } else {
+    // Deployed piggy bank smart contract with ethers library
+    piggyBankContract = ethersPiggyBankContract;
+    // Deployed hst smart contract with ethers library
+    hstContract = ethersHstContract;
+    // Deployed collectibles smart contract with ethers library
+    collectiblesContract = ethersCollectiblesContract;
+    // Deployed failing smart contract with ethers library
+    failingContract = ethersFailingContract;
+  }
+
+  // Changing buttons and label state when piggy bank contract is deployed
+  if (piggyBankContract) {
+    contractStatus.innerHTML = 'Deployed';
+    depositButton.disabled = false;
+    withdrawButton.disabled = false;
+  } else {
+    contractStatus.innerHTML = 'Not clicked';
+    depositButton.disabled = true;
+    withdrawButton.disabled = true;
+  }
+
+  // Changing buttons and label state when collectibles contract is deployed
+  if (collectiblesContract) {
+    collectiblesStatus.innerHTML = 'Deployed';
+    mintButton.disabled = false;
+    mintAmountInput.disabled = false;
+  } else {
+    collectiblesStatus.innerHTML = '';
+    mintButton.disabled = true;
+    mintAmountInput.disabled = true;
+  }
+
+  // Changing buttons and label state when hst contract is deployed
+  if (hstContract) {
+    tokenAddress.innerHTML =
+      libraryInUse === 'web3'
+        ? hstContract.options.address
+        : hstContract.address;
+    watchAsset.disabled = false;
+    transferTokens.disabled = false;
+    approveTokens.disabled = false;
+    transferTokensWithoutGas.disabled = false;
+    approveTokensWithoutGas.disabled = false;
+  } else {
+    tokenAddress.innerHTML = '';
+    watchAsset.disabled = true;
+    transferTokens.disabled = true;
+    approveTokens.disabled = true;
+    transferTokensWithoutGas.disabled = true;
+    approveTokensWithoutGas.disabled = true;
+  }
+
+  // Changing buttons and label state when failing contract is deployed
+  if (failingContract) {
+    failingContractStatus.innerHTML = 'Deployed';
+    sendFailingButton.disabled = false;
+  } else {
+    failingContractStatus.innerHTML = 'Not clicked';
+    sendFailingButton.disabled = true;
+  }
+
+  console.log(libraryInUse);
```

Comment:
This console log can be removed.

Reply:
Fixed in: 91555c04c8088ee472c87cc25843642994ec92b4